notebooks/cleaning_data.ipynb

```diff
@@ -0,0 +1,2892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    prefixes = ['license', 'tnc_user', 'tnc_use_', 'bike_', 'no_bike_nyc_', 'bike_share_', 'disability_', 'no_bike_share_', 'race_', 'no_bike_share_']
</pre><p>'no_bike_share_' is duplicated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Clustering-Analysis/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>